src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -57,4 +99,51 @@ protected function formatForTable(array $sprykDefinitions): array
 
         return $formatted;
     }
+
+    /**
+     * @param array $sprykDefinitions
+     *
+     * @return array
+     */
+    protected function formatOptionsTable(array $sprykDefinitions): array
+    {
+        $formatted = ['mode' => ['mode']];
+        foreach ($sprykDefinitions as $option => $optionDefinition) {
+            if (isset($optionDefinition['value'])) {
+                continue;
+            }
+
+            $formatted[$option] = [$option];
+        }
+        sort($formatted);
+
+        return $formatted;
+    }
+
+    /**
+     * @param array $sprykDefinition
+     *
+     * @return array
+     */
+    protected function getOptionalSpryks(array $sprykDefinition): array
+    {
+        $optionalSpryks = [];
+        /** @var array $postAndPreSpryks */
+        $postAndPreSpryks = ($sprykDefinition['preSpryks'] ?? []) + ($sprykDefinition['postSpryks'] ?? []);
```

Comment:
This should be moved into a separate method to get rid of the inline comment and the rather unreadable line in here.

Reply:
You could even add a proper filter into the extracted method to get rid of the is_array check in the loop.

Reply:
Done.